app/js/lib/intent-parser.js

```diff
@@ -1,46 +1,110 @@
 'use strict';
 
 /*
-Examples of phrases to detect and parse:
-[Remind] me [to] pick Sasha from Santa Clara University [at] 5PM today.
-[Remind] me [that] it is picnic day [on] July 4th.
-[Remind] us [to] go to the opera [at] 7:15pm on 2nd February.
-[Remind] us [to] go at my mum's [at] 11:30am on 31st July.
-[Remind] everybody [to] pack their stuff [by] next Friday 5pm.
+Examples of supported phrases:
+Remind me to pick Sasha from Santa Clara University at 5PM today.
+Remind me that it is picnic day on July 4th.
+Remind us to go to the opera at 7:15pm on 2nd February.
+Remind us to go at my mum's at 11:30am on 31st July.
+Remind everybody to pack their stuff by next Friday 5pm.
+Remind me that every Tuesday night is trash day.
+
+To add:
+Remind me every Tuesday to take the bin out.
+Remind Guillaume on Thursdays evening to go to his drawing class.
+Remind me that I should prepare my appointment tomorrow morning.
 */
 
-const pattern = {
-  regexp: new RegExp('^Remind\\b([^to]+)\\bto\\b(.+)\\bat\\b(.+)$', 'iu'),
-  placeholders: {
-    user: 0,
-    action: 1,
-    time: 2,
+const PATTERNS = {
+  en: {
```

Comment:
We could use BCP 47 language tags for the language keys and then set the same value in the Speech Recognition API.